### DISCUSSION TITLE
App won't open

### DISCUSSION THREAD

**Discussion by cyberrin** [YerongAI/Office-Tool#495] [Q&A] [answered]
I have .net desktop runtimes installed, when I try to open otp it appears for half a second in the task manager and disappears.
What's wrong?

**Comment by YerongAI** [comment]
Can you provide your OS information?

**Reply by cyberrin** [reply]
win 7 sp1 x64

**Reply by YerongAI** [reply]
> win 7 sp1 x64

Please install [Update Pack](https://download.coolhub.top/Extensions/Win7_UpdatePack/) first, then try again.

**Reply by cyberrin** [reply]
hmm, ill try it now

**Reply by cyberrin** [reply]
unfortunately, still won't start

**Reply by YerongAI** [reply]
> unfortunately, still won't start

You can open event viewer, select _Windows Event_ -> _Applications_, then find out which is .NET problems and contains Office Tool Plus, send it to me please.

**Reply by cyberrin** [reply]
> > unfortunately, still won't start
> 
> You can open event viewer, select _Windows Event_ -> _Applications_, then find out which is .NET problems and contains Office Tool Plus, send it to me please.

Description: A .NET application failed.
Application: Office Tool Plus.exe
Path: C:\Users\cyberrin\Downloads\Office_Tool_with_runtime_v8.3.8.0\Office Tool\Office Tool Plus.exe
Message: Failed to load the dll from [C:\Program Files (x86)\dotnet\host\fxr\6.0.4\hostfxr.dll], HRESULT: 0x80070057
The library hostfxr.dll was found, but loading it from C:\Program Files (x86)\dotnet\host\fxr\6.0.4\hostfxr.dll failed
  - Installing .NET prerequisites might help resolve this problem.

**Reply by cyberrin** [reply]
look

**Reply by YerongAI** [reply]
> look

Seems like you have runtime installed on your machine? What about try to uninstall it?

**Reply by cyberrin** [reply]
I tried several times, it does not help, I even wonder what is wrong

**Reply by YerongAI** [reply]
> I tried several times, it does not help, I even wonder what is wrong

Seems like you don't have all needed updates installed, does UpdatePack7R2 work correctly? Or you can [download and install update](https://www.catalog.update.microsoft.com/Search.aspx?q=KB4457144) manually.

**Reply by cyberrin** [reply]
> > I tried several times, it does not help, I even wonder what is wrong
> 
> Seems like you don't have all needed updates installed, does UpdatePack7R2 work correctly? Or you can [download and install update](https://www.catalog.update.microsoft.com/Search.aspx?q=KB4457144) manually.

well, actually 2 updates could not be installed .. I really thought that it was not critical, I will try to install them separately today

**Reply by cyberrin** [reply]
during reboot the system said failed to configure updates, I also tried to install them several times, but the result is the same

**Reply by YerongAI** [reply]
> during reboot the system said failed to configure updates, I also tried to install them several times, but the result is the same

You may need to fix your Windows using `dism` command or `sfc /scannow`. For more information you can search on Google.

**Reply by cyberrin** [reply]
nothing works, I will reinstall Windows, I hope the problem will be solved

**Reply by cyberrin** [reply, marked as answer]
And of course, reinstalling Windows helped. thanks for your help and your time.
